bin/docs.js

```diff
@@ -47,7 +47,7 @@ function getDocsContent(themeName) {
       if (t.colorKeys.indexOf(key) !== -1) {
         if (value && value.indexOf('#') !== -1) {
           let c = value.split('#')[1]
-          md += `| ${childKey} | ${value} | [![${value}](https://via.placeholder.com/32x16/${c}/000000?text=+)](https://www.colorhexa.com/${c}) |\n`
```

Comment:
What was all this?

Reply:
Just another way that Dave set up the colors. I think Github changed the way they process urls in Readme's and it broke it recently.